Fabric/Fastfile

```diff
@@ -0,0 +1,233 @@
+        load_keys
+
+        deliver
+    end
+
+    lane :testflight do |options|
+        load_keys
+
+        ensure_git_branch(
+            branch: "(master|release\\S+|hotfix\\S+)"
+        )
+
+        ensure_git_status_clean
+
+        verify_xcode_version
+
+        version_number = options[:version_number]
+
+        raise "Missing version number: use version_number parameter" unless version_number != nil
+
+        increment_version_number(version_number: version_number)
+        build_number = increment_build_number
+        complete_version_number = version_string(version_number, build_number)
+
+        commit_version_bump(
+            message: "Version bump to #{complete_version_number}",
+            xcodeproj: "Fabric.xcodeproj"
+        )
+
+        change_log = "CHANGELOG TBD. Version: #{complete_version_number}"
+
+        skip_metadata = options[:skip_metadata]
+
+        unless skip_metadata == true
+            clear_derived_data
+            screenshots
+            update_metadata
+        else
+            Helper.log.info("Skipping taking screenshots and uploading metadata".green)
+        end
+
+        clean_build_artifacts
+        clear_derived_data
+
+        sigh(output_path: "/tmp")
+        gym(
+            scheme: "Fabric",
+            configuration: "Release",
+            clean: true
+        )
+
+        testflight(
+            changelog: change_log,
+            skip_submission: true,
+            distribute_external: false
+        )
+
+        set_github_release(
+            repository_name: "crashlytics/Fabric-iOS",
+            api_token: ENV["GITHUB_API_TOKEN"],
+            name: complete_version_number,
+            tag_name: tag_name("testflight", version_number, build_number),
+            commitish: git_branch,
+            description: change_log_since_last_tag,
+            is_prerelease: false
+        )
+
+        slack(
+            message: "Submitted build of Fabric to TestFlight: v#{complete_version_number}",
+            channel: "#general"
+        )
+
+        rocket
+    end
+
+    lane :carthage do |options|
+        verify_xcode_version
+
+        carthage(
+            command: options[:update] ? "update" : "bootstrap",
+            platform: "iOS",
+            use_binaries: false
+        )
+    end
+
+    lane :code_gen do
+        root_directory = "#{File.expand_path(File.dirname(__FILE__))}/../"
+
+        sdk_path = sh("xcodebuild -version -sdk iphoneos9.2 Path")
+
+        sh("#{root_directory}/Pods/R.swift/rswift -p \"#{root_directory}/Fabric.xcodeproj\" -t Fabric --bundleIdentifier io.fabric.ios --productModuleName Fabric --buildProductsDir \"#{root_directory}\" --developerDir \"#{root_directory}\" --sourceRoot \"#{root_directory}\" --sdkRoot \"#{sdk_path}\" \"#{root_directory}/Fabric/\"")
+    end
+
+    private_lane :rocket do
+        puts "
+        /\\
+       |==|
+       |  |
+       |  |
+       |  |
+      /____\\
+      |    |
+      |fab |
```

Comment:
:heart: This should be a built-in action, please submit a separate PR to fastlane adding this :rocket: